fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -162,15 +150,14 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
```

Comment:
I may be reading this wrong, but if `describedby` is `null`, isn't `describedby.iterator()` going to get executed anyway, throwing an NPE? That's based on what I can get out of https://docs.oracle.com/javase/specs/jls/se7/html/jls-14.html#jls-14.14.1.1